src/js/components/DataTable/index.d.ts

```diff
@@ -105,7 +105,9 @@ export interface DataTableProps<TRowType = any> {
   step?: number;
 
   // Events
-  onClickRow?: (event: MouseClick<TRowType> | KeyPress<TRowType>) => void;
+  onClickRow?:
+    | 'onSelect'
```

Comment:
The term "onSelect" seems a little odd to me here. Like it will call any `onSelect` handler when the row is clicked. I'm wondering if we should use "select" here, indicating that if the user clicks the row, it will be treated as initiating a selection.

Reply:
that's fair and makes sense